.github/workflows/build_package.yml

```diff
@@ -66,6 +69,7 @@ jobs:
         with:
           path: 'main_checkout'
           submodules: true
+          ref: ${{ github.event.inputs.commit }}
```

Comment:
Maybe add a default here?

Reply:
I set an empty default as an input to the action (line 26), is there a different way to set the default at this line?

Reply:
(When an empty value is passed in for the ref, it just uses the latest commit - tested here https://github.com/not-jenni/iree/runs/6891334495?check_suite_focus=true)